src/DIRAC/FrameworkSystem/Service/TokenManagerHandler.py

```diff
@@ -1,57 +1,75 @@
-""" TokenManager service
+"""TokenManager service responsible for token management, namely storing, updating,
+requesting new tokens for DIRAC components that have the appropriate permissions.
```

Comment:
is it only for DIRAC components, or for users too?

Reply:
Also mention that there's no dips version

Reply:
>is it only for DIRAC components, or for users too?

yes, for users too. Upon successful authorization, users receive tokens from `TokenManager`, which in turn receives them from third-party IdPs.

> Also mention that there's no dips version

I think we stop supporting DIPS, but OK, I added DIPS support

Reply:
no, we want to remove dips, just say it explicitly that there is no dips support

Reply:
Apparently I misunderstood, added a note about it